prov/efa/src/rxr/rxr_ep.c

```diff
@@ -1065,44 +1065,21 @@ static ssize_t rxr_ep_cancel(fid_t fid_ep, void *context)
  */
 static int efa_set_fi_hmem_p2p_opt(struct efa_ep *efa_ep, int opt)
 {
-	struct efa_hmem_info *hmem_info;
+	int i, ret;
```

Comment:
`ret` -> `err`

Reply:
`s/ret/err/g` makes sense when the value of `efa_hmem_validate_p2p_opt()` is only returned in a non-successful case, but as it stands, we also return the value from `efa_hmem_validate_p2p_opt()` if it's `FI_SUCCESS`

Reply:
FI_SUCESS is 0, which is no error

Reply:
That's my point. Having `return err` when `err` can be `FI_SUCCESS` isn't quite accurate.

Reply:
It increase readability.  with `err = xxx`.  `if (!err)` means "if there is no error". Meanwhile `if (!ret)` is hard to understand.

This is widely used convention throughout the code base.